### PR TITLE
✨ RW-23: Add time zone choice to admin panel

### PR DESCRIPTION
**Pull request by kasia-antos**

### CODE REVIEW

File: backend/replant/admin/tree.py [hunk 1]
```diff
@@ -52,10 +52,6 @@ class TreeAdmin(TrackableModelAdmin):
         "minting_state",
     ]
 
-    def get_queryset(self, request):
-        qs = super().get_queryset(request)
-        return qs.select_related("sponsor", "created_by", "species")
```
Comment (tulustul) [inline backend/replant/admin/tree.py:57]: Why removing the `select_related`?

Comment (tulustul) [inline backend/replant/admin/tree.py:57]: Why removing the `select_related`?

Comment (kasia-antos) [reply backend/replant/admin/tree.py:57]: It's doubled now, I added this in separate PR

File: backend/replant/models/user.py [hunk 2]
```diff
@@ -13,3 +17,4 @@ class User(AbstractUser):
         on_delete=models.PROTECT,
         verbose_name="planting organization / community",
     )
+    time_zone = models.CharField(max_length=40, default="UTC")
```
Comment (tulustul) [inline backend/replant/models/user.py:20]: choices?

File: backend/replant/models/user.py [hunk 3]
```diff
@@ -2,8 +2,12 @@
 from django.db import models
 from phonenumber_field.modelfields import PhoneNumberField
 
+from replant.time_zones import TIME_ZONES
+
 
 class User(AbstractUser):
+    TIME_ZONES = TIME_ZONES
```
Comment (tulustul) [inline backend/replant/models/user.py:9]: Is it just to make it easily available in the template?

File: backend/replant/templates/admin/base_site.html [hunk 4]
```diff
@@ -21,6 +21,40 @@
...
+    font-weight: 400;
+    font-size: 0.6875rem;
+    letter-spacing: 0.5px;
+    color: var(--header-link-color);
+    background-color: var(--breadcrumbs-bg);
+    border-color: var(--breadcrumbs-bg)
+  }
+
+  @media (max-width: 1024px) {
+    div #user-tools {
```
Comment (tulustul) [inline backend/replant/templates/admin/base_site.html:45]: `div` is pointless here

File: backend/replant/templates/admin/base_site.html [hunk 5]
```diff
@@ -21,6 +21,40 @@
...
+    }
+
+    #time-zone {
+      padding: 0;
+      font-weight: 400;
+    }
+
+    #time-zone label {
+      font-size: 0.6875rem;
+    }
```
Comment (tulustul) [inline backend/replant/templates/admin/base_site.html:56]: Maybe on desktop
```css
#user-tools {
  flex: 1;
  margin-right: 15px;
}
```

**Review by tulustul — APPROVED** [review]

File: backend/replant/models/user.py (hunk 3 above)
Comment (kasia-antos) [reply backend/replant/models/user.py:9]: yep

File: backend/replant/admin/tree.py (hunk 1 above)
Comment (kasia-antos) [reply backend/replant/admin/tree.py:57]: It's doubled now, I added this in separate PR

File: backend/replant/templates/admin/base_site.html (hunk 4 above)
Comment (kasia-antos) [reply backend/replant/templates/admin/base_site.html:45]: Hm otherwise I'm not able to overwrite this 🤔 

image

**Pull request merged**